qiskit/circuit/quantumcircuit.py

```diff
@@ -3192,10 +3192,12 @@ def draw(
                 merges the lines generated by the `text` output so the drawing
                 will take less vertical room.  Default is ``medium``. Only used by
                 the ``text`` output, will be silently ignored otherwise.
-            idle_wires: Include idle wires (wires with no circuit elements)
-                in output visualization. Default is ``False`` unless the
+            idle_wires: Include (or not) idle wires (wires with no circuit elements)
+                in output visualization. The string ``"auto"`` is also possible, in which
+                case idle wires are show except that the circuit has a layout attached.
+                Default is ``"auto"`` unless the
                 user config file (usually ``~/.qiskit/settings.conf``) has an
-                alternative value set. For example, ``circuit_idle_wires = True``.
+                alternative value set. For example, ``circuit_idle_wires = False``.
```

Comment:
Do we need to update the configuration-file logic to make it possible to specify `"auto"` in there too?